babel.config.js

```diff
@@ -0,0 +1,19 @@
+const presets = [
```

Comment:
Can you clarify why we need babel?

Reply:
Sorry, this is from having set up the project based on an earlier one of mine. I understand there is no need for it. I'll remove it.